docs/source/troubleshooting/logs.rst

```diff
@@ -95,19 +95,15 @@ The following list outlines some of the common logs and what methods are recomme
 Note that when writing to these logs that it’s important to be aware of how they behave.
 For example, by default the ``errlog.txt`` file will persist across reboots but be rotated by ``logrotate`` very quickly due to file size while the ``messages`` log will not persist across reboots but will be rotated less often.
 
-Enabling Persistent Logs
-========================
-By default, logs are stored in ``/var/log`` which is a symlink to ``/var/volatile/log``.
-The ``/var/volatile`` location is a RAM-based tmpfs filesystem whose contents are lost on each reboot.
-This section explains how to make those logs persistent.
+Enabling Persistent Logs (NILRT <= 11.1)
+========================================
 
-**Warning:** Performing the steps in this section can impact system performance.
```

Comment:
If you retain this section, use a `.. warning` directive.

Reply:
I am going to drop this as we haven't observed any performance impact so far and the original warning was most likely speculative rather than data driven.